pygmt/tests/data/focal_mechanisms.txt

```diff
@@ -0,0 +1 @@
+-127.43 40.81 12 -3.19 1.16, 3.93, -1.02, -3.93, -1.02, 23
```

Comment:
This file is only used in one test. It's better to write the focal mechanism to a temporary file, plot it and then remove it.

Reply:
Addressed in b173b49